examples/advanced/main.go

```diff
@@ -26,29 +26,15 @@ import (
 	_ "github.com/wasilibs/nottinygc"
 
 	"sigs.k8s.io/kube-scheduler-wasm-extension/examples/advanced/plugin"
-	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/config"
 	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/enqueue"
-	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/klog"
 	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/prescore"
 	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/score"
 )
 
 // main is compiled to an exported Wasm function named "_start", called by the
 // Wasm scheduler plugin during initialization.
 func main() {
-	// The plugin package uses only normal Go code, which allows it to be
-	// unit testable via `tinygo test -target=wasi` as well normal `go test`.
-	//
-	// The real implementations, such as `config.Get()` use Wasm host functions
-	// (go:wasmimport), which cannot be tested with `tinygo test -target=wasi`.
-	plugin, err := plugin.New(klog.Get(), config.Get())
```

Comment:
this line prevented the direct plumbing of things that would interfere with tests, specifically binding to `klog.Get(), config.Get()`. If something is broke, look here. Advanced was how to show how to unit test inside tinygo, and likely this is still needed.

Reply:
@codefromthecrypt 
Thanks for your comments!
Just clarify, shouldn't I remove this line?
In my implementation, klog.Get() and config.Get() are called within plugin.go, is it not good implementaiton?

Reply:
Thanks for your feedback! Reverted this line.